src/callbacks/charts.py

```diff
@@ -194,7 +194,7 @@ def create_bar_chart(avg_data, spend_metric, spend_metric_label):
 
     bar_chart = alt.Chart(avg_data, width='container', height=305).mark_bar(color="teal").encode(
         x=alt.X(f'mean({spend_metric}):Q', title=spend_metric_label),
-        y=alt.Y('name:N', title="Country", sort='x'),  
+        y=alt.Y('name:N', title=None, sort='x'),  
```

Comment:
oh I mean for example here, I wonder if we could just say `y=alt.Y('name:N', sort='x')`